src/src/luaParams.cpp

```diff
@@ -68,4 +68,44 @@ struct tagLuaItem_command luaWebUpdate = {
     LUA_COMMAND_SIZE(luaWebUpdate)
 };
 #endif
+
+//----------------------------VTX ADMINISTRATOR------------------
+
+struct tagLuaItem_textSelection luaVtxBand = {
+    {0,(uint8_t)CRSF_TEXT_SELECTION},//id,type
+    "Band",
+    "Off;A;B;E;F;R;L",
+    {0,0,6},//value,min,max
+    emptySpace,
+    LUA_TEXTSELECTION_SIZE(luaVtxBand)
+};
+
+struct tagLuaItem_uint8 luaVtxChannel = {
+    {0,(uint8_t)CRSF_UINT8},//id,type
+    "Channel",
+    {0,0,7},//value,min,max
```

Comment:
I think this needs to be text 1;2;3;4;5;6;7;8 as thats what users expect for VTX channels. If we do that can we set min as 1 max as 8 and does the LUA script adjust accordingly?

Reply:
yes we can set min as 1 and max as 8, leave default as 0 as default value is not checked and used. I also just did a test to confirm this and its good. will update if there is no other consideration to use uint8?

Reply:
Looks like channel 1 is converted to value 0, so leaving it as a list of strings is probably a better solution rather than having an adjustment in the MSP function.

Reply:
I might reorder Band as well and put Off at the end.  This just cleans up the Idx calc later for the MSP packet.